modules/search/instant-search/lib/tracks.js

```diff
@@ -0,0 +1,34 @@
+/**
```

Comment:
Yes, namely: bundle size. Importing `client/lib/analytics/index.js` (which is an outdated copy of Calypso's Tracks library) would significantly bloat the Instant Search payload. This should be pretty obvious by comparing LOC (234 vs 34).

Reply:
Here's another way to think about it: what if every block and feature that needs frontend tracking would copy paste this code around? 

Would it be possible to refactor the original bit in Jetpack so that it would not explode the bundle?

Reply:
> what if every block and feature that needs frontend tracking would copy paste this code around?

It wouldn't be ideal, but I could imagine worse; at least it's only 21 LOC 😄

Nonetheless, if we were to refactor `client/lib/analytics/index.js`, I'd recommend removing all external dependencies and splitting the library into three parts:
 
- Tracks analytics
- MC analytics
- Google Analytics

What do you think?